### PR TITLE
Czech translation

### PR DESCRIPTION
**Pull request by wociscz**
Hi, I've translated fluidd GUI into Czech language.

Tested locally, seems to works as expected. Some keys are missing and can't be translated via yaml yet - for example
* settings "add category"
* main window - user icon "manage accounts"
* idle/ready in the printer status header

Best Regards!

image

image

### CODE REVIEW

**Comment by Afterschnitzel** [comment]
Please translate "extruse" as "extruze"
"Extruse" sounds pretty arcahic.

File: src/locales/cz.yaml [hunk 1]
```diff
@@ -0,0 +1,448 @@
...
+  gcode:
+    btn:
+      load_current_file: Načíst aktuální soubor
+    label:
+      current_layer_height: Současná výška vrstvy
+      follow_progress: Sledovat průběh tisku
+      layer: Vrstva
+      layers: Vrstvy
+      parsed: Analyzována(y)
+      show_extrusions: Zobrazit extrusi
```
Comment (vrbalu) [inline src/locales/cz.yaml:106]:
```suggestion
      show_extrusions: Zobrazit extruzi
```

File: src/locales/cz.yaml [hunk 2]
```diff
@@ -0,0 +1,448 @@
...
+      system: Systém
+      system_overview: Informace o systému
+      temperature: Teploty
+      tool: Nástroj
+      tune: Ladění
+    tooltip:
+      estop: Nouzové zastavení
+      reload_klipper: Obnoví konfiguraci klipperu.
+      reload_restart_klipper: Obnoví konfiguraci klipperu a restartne MCU.
+      restart_klipper: Restartne systémovou službu klipper.
```
Comment (vrbalu) [inline src/locales/cz.yaml:310]:
```suggestion
      restart_klipper: Restartuje systémovou službu klipper.
```

File: src/locales/cz.yaml [hunk 3]
```diff
@@ -0,0 +1,448 @@
...
+      stats: Statistiky tiskárny
+      system: Systém
+      system_overview: Informace o systému
+      temperature: Teploty
+      tool: Nástroj
+      tune: Ladění
+    tooltip:
+      estop: Nouzové zastavení
+      reload_klipper: Obnoví konfiguraci klipperu.
+      reload_restart_klipper: Obnoví konfiguraci klipperu a restartne MCU.
```
Comment (vrbalu) [inline src/locales/cz.yaml:309]:
```suggestion
      reload_restart_klipper: Obnoví konfiguraci klipperu a restartuje MCU.
```

File: src/locales/cz.yaml [hunk 4]
```diff
@@ -0,0 +1,448 @@
...
+  history:
+    msg:
+      confirm: Jste si jistí? Toto smaže veškerou historii a statistiky tiskárny
+  printer:
+    state:
+      busy: Zaneprázdněn
+      complete: Kompletní
+      idle: Nečinný
+      loading: Nahrávání
+      paused: Pausa
```
Comment (vrbalu) [inline src/locales/cz.yaml:320]:
```suggestion
      paused: Pauza
```

File: src/locales/cz.yaml [hunk 5]
```diff
@@ -0,0 +1,448 @@
...
+      total_time: Celkový čas
+      total_time_avg: Průměr na tisk
+      uncategorized: Bez kategorie
+      unretract_extra_length: Extra délka detrakce
+      unretract_speed: Rychlost detrakce
+      used: použito
+      variance: Rozptyl
+      velocity: Rychlost
+      visible: Viditelné
+      z_offset: Ofset Z
```
Comment (vrbalu) [inline src/locales/cz.yaml:240]:
```suggestion
      z_offset: Odchylka Z
```

File: src/locales/cz.yaml [hunk 6]
```diff
@@ -0,0 +1,448 @@
...
+      product_name: Název produktu
+      serial_number: Seriové číslo
+      system_load: Zátěž systému
+      system_memory: Paměť
+      system_utilization: Využití systému
+      total_memory: Paměti celkem
+  tool:
+    btn:
+      home_x: X
+      home_y: 'Y'
```
Comment (vrbalu) [inline src/locales/cz.yaml:418]: Unify, either 'X' and 'Y' or X and Y

File: src/locales/cz.yaml [hunk 7]
```diff
@@ -0,0 +1,448 @@
...
+    label:
+      auto_scroll: Automatický posun
+      hide_temp_waits: Skrýt čekání na teplotu
+    placeholder:
+      command: '''tab'' pro autodoplňování, ''help'' pro příkazy ''šipky'' pro historii'
+  endpoint:
+    error:
+      cant_connect: >-
+        Něco se pokazilo a fluidd se nemůže připojit na cílovou adresu.
+        Jste si jistí že je adresa správně zadaná?
```
Comment (Afterschnitzel) [inline src/locales/cz.yaml:45]: Jste si jistý, že je adresa správně zadaná?

File: src/locales/cz.yaml [hunk 8]
```diff
@@ -0,0 +1,448 @@
...
+      cors_error: Zablokováno CORS politikou
+      cors_note: >-
+        Což znamená, že bude třeba upravit konfiguraci moonrakeru. Prosím
```
Comment (Afterschnitzel) [inline src/locales/cz.yaml:48]: To znamená, že bude třeba upravit konfiguraci moonrakeru.

File: src/locales/cz.yaml [hunk 9]
```diff
@@ -0,0 +1,448 @@
...
+    label:
+      dir_name: Název adresáře
+      disk_usage: Využití disku
+      diskinfo: Informace o disku
+      downloaded: Staženo
+      file_name: Název souboru
+      transfer_rate: Přenosová rychlost
+      uploaded: Nahráno
+    msg:
+      confirm: Jste si jisti? Smažete tím všechny soubory a adresáře.
```
Comment (Afterschnitzel) [inline src/locales/cz.yaml:81]: Jste si jistý?

File: src/locales/cz.yaml [hunk 10]
```diff
@@ -0,0 +1,448 @@
...
+      parsed: Analyzována(y)
+      show_extrusions: Zobrazit extrusi
+      show_moves: Zobrazit pohyby
+      show_next_layer: Následující vrstva
+      show_previous_layer: Předchozí vrstva
+      show_retractions: Zobrazit retrakce
+    msg:
+      confirm: >-
+        Soubor "%{filename}" má %{size}, může být dost náročný na prostředky
+        Vašeho systému. Jste si jistí?
```
Comment (Afterschnitzel) [inline src/locales/cz.yaml:114]: Jste si jistý?

File: src/locales/cz.yaml [hunk 11]
```diff
@@ -0,0 +1,448 @@
...
+        arrayofnums: Pouze čísla
+        exists: Již existuje
+        invalid_url: Neplatná URL
+        max: Max %{max}
+        min: Min %{min}
+        min_or_0: Min %{min} nebo 0
+        password_username: Nemůžu najít uživatele
+        required: Vyžadováno
+      msg:
+        confirm: Jste si jistí?
```
Comment (Afterschnitzel) [inline src/locales/cz.yaml:255]: Jste si jistý?

File: src/locales/cz.yaml [hunk 12]
```diff
@@ -0,0 +1,448 @@
...
+        confirm_reboot_host: Jste si jistí? Toto restartuje váš system.
```
Comment (Afterschnitzel) [inline src/locales/cz.yaml:256]: Jste si jistý?

File: src/locales/cz.yaml [hunk 13]
```diff
@@ -0,0 +1,448 @@
...
+        confirm_shutdown_host: Jste si jistí? Toto vypne váš system.
```
Comment (Afterschnitzel) [inline src/locales/cz.yaml:257]: Jste si jistý?

File: src/locales/cz.yaml [hunk 14]
```diff
@@ -0,0 +1,448 @@
...
+      tool: Nástroj
+      tune: Ladění
+    tooltip:
+      estop: Nouzové zastavení
+      reload_klipper: Obnoví konfiguraci klipperu.
+      reload_restart_klipper: Obnoví konfiguraci klipperu a restartne MCU.
+      restart_klipper: Restartne systémovou službu klipper.
+  history:
+    msg:
+      confirm: Jste si jistí? Toto smaže veškerou historii a statistiky tiskárny
```
Comment (Afterschnitzel) [inline src/locales/cz.yaml:313]: Jste si jistý?

File: src/locales/cz.yaml [hunk 15]
```diff
@@ -0,0 +1,448 @@
...
+      printing: Tiskne
+      ready: Připraven
+      standby: Pohotovostní
+  setting:
+    btn:
+      add_camera: Přidat kameru
+      add_thermal_preset: Přidat přednastavení
+      add_user: Přidat uživatele
+      reset: Resetovat
+      select_theme: Vybrat téma
```
Comment (Afterschnitzel) [inline src/locales/cz.yaml:330]: Vybrat motiv

File: src/locales/cz.yaml [hunk 16]
```diff
@@ -0,0 +1,448 @@
...
+      check_for_updates: Zkontrolovat aktualizace
+      finish: Hotovo
+      update: Aktualizovat
+      view_versions: Zobrazit verze
+    label:
+      commit_history: Historie commitů
+      commits_on: Commity zapnout
+      committed: Commitnuto
+      dirty: DIRTY
+      invalid: INVALID
```
Comment (Afterschnitzel) [inline src/locales/cz.yaml:434]: NEPLATNÝ

File: src/locales/cz.yaml [hunk 17]
```diff
@@ -0,0 +1,448 @@
...
+    btn:
+      check_for_updates: Zkontrolovat aktualizace
+      finish: Hotovo
+      update: Aktualizovat
+      view_versions: Zobrazit verze
+    label:
+      commit_history: Historie commitů
+      commits_on: Commity zapnout
+      committed: Commitnuto
+      dirty: DIRTY
```
Comment (Afterschnitzel) [inline src/locales/cz.yaml:433]: ZNEČIŠTĚNÝ

File: src/locales/cz.yaml [hunk 18]
```diff
@@ -0,0 +1,448 @@
...
+      invalid: INVALID
+      os_packages: Balíčky systému
+      package_list: Seznam balíčků
+      up_to_date: AKTUÁLNÍ
+      updates_available: Dostupné aktualizace 
+    status:
+      finished: Aktualizace hotova
+      updating: Aktualizuju...
```
Comment (Afterschnitzel) [inline src/locales/cz.yaml:441]: Aktualizuji

File: src/locales/cz.yaml [hunk 19]
```diff
@@ -0,0 +1,448 @@
+app:
+  bedmesh:
+    label:
+      box_scale: Měřítko boxu
+      flat_surface: Zobrazit plochou rovinu
+      mesh_matrix: Mesh matice
```
Comment (Afterschnitzel) [inline src/locales/cz.yaml:6]: Síťová matice

File: src/locales/cz.yaml [hunk 20]
```diff
@@ -0,0 +1,448 @@
...
+      profile_name: Název profilu
+      remove_profile: Odstranit %{name} profil
+      scale: Barevná škála
+      wireframe: Drátěný model
+    msg:
+      hint: >-
+        Pokud uložíte pod jiným názvem než %{name}, 
+        můžete původní profil %{name} odstranit.
+      not_found: Žádné meshe nenalezeny.
+      not_loaded: Žádný mesh nenačten.
```
Comment (Afterschnitzel) [inline src/locales/cz.yaml:17]: Síť bodů nenačtena

**Review by vrbalu — APPROVED** [review]
Looks good to me as to native Czech speaker. :+1:

**Review by dweight — APPROVED** [review]

**Pull request merged**